### PR TITLE
Upgrade gradle and kotlin version

### PR DESCRIPTION
**Pull request by heralight**

### CODE REVIEW

File: sample-kotlin/build.gradle [hunk 1]
```diff
@@ -25,7 +25,6 @@ android {
     }
   }
   sourceSets {
-    main.java.srcDirs += 'src/main/kotlin'
```
Comment (sockeqwe) [inline sample-kotlin/build.gradle]: Are you sure that the kotlin sample compiles without that?

Comment (heralight) [inline sample-kotlin/build.gradle]: You have right,  strange thing, I don't change this line... Perhaps in android studio.
I roll back this line.

**Comment by sockeqwe** [comment]
Thanks!

**Comment by heralight** [comment]
It's  me, thank you! It's a great project !

**Pull request merged**